src/redux/actions/pipeline/runGem2s.js

```diff
@@ -34,6 +35,10 @@ const runGem2s = (experimentId) => async (dispatch, getState) => {
         headers: {
           'Content-Type': 'application/json',
         },
+        body: JSON.stringify({
+          shouldRun: gem2sHash !== oldGem2sParams,
+          gem2sHash,
+        }),
```

Comment:
why passing this at all? Can we simply assume that we only call the API if we have to run gem2s, otherwise we don't. Why does the API need to know about this gem2s hash?

Reply:
You're right. In the cases where we want to run GEM2S, we still need to pass the hash. I'll change it accordingly.